src/inline-alert.test.visuals.ts

```diff
@@ -7,6 +7,22 @@ for (const story of stories['Inline Alert']) {
   test.describe(story.id, () => {
     for (const theme of story.themes) {
       test.describe(theme, () => {
+        test(':focus', async ({ page }, test) => {
+          await page.goto(`?id=${story.id}&globals=theme:${theme}`);
+
+          await page
+            .locator('glide-core-inline-alert')
+            .evaluate<void, GlideCoreInlineAlert>((element) => {
+              element.removable = true;
+            });
+
+          await page.getByRole('button').focus();
```

Comment:
A case where  `getByRole()` instead of `locator('glide-core-icon-button')` makes sense because the latter would means this test has knowledge of Inline Alert's internals (that it uses an Icon Button), and so would be slightly brittle. 

This test shouldn't have to know how Inline Alert implements its button and whether it's a `<button>` or a `<glide-core-icon-button>`.

Reply:
I dig it!